<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kodowanie Hamminga</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="hammingInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="errorButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wektor Błędu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detekcjaButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detekcja</string>
    </property>
   </widget>
   <widget class="QPushButton" name="korekcjaButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Korekcja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Podaj binararną informacje do zakodowania</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encodeButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Koduj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generuje losowy ciąg binarny&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generuje losowy ciąg binarny&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Generuj Losowo</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Skorygowany">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>      Wektor błędu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Skorygowany kod </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="kodHamminga">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="l2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
color: rgb(255,153,153);
font: 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>460</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
color: rgb(178,255,102);
font: 9pt &quot;Arial Black&quot;;
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Detekcja błędu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
color: rgb(153,153,255);
font: 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>- pozycja skorygowanego błędu</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="HammingWithError">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Kod Hamminga</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="WektorBledu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearOutHam">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>350</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Czyści output&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>- pozycja błędu</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DetekcjaB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LEGENDA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kod Hamminga z błędem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>- miejsca bitów parzystości</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pokazG">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaż macierz G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pokazH">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaż macierz H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pokazSposob">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaż sposób detekcji bledu</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="syndrom">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>      Syndrom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLista">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista kodów </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Podaj binararną informacje do dekodowania</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dekodujInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="decodeButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Dekoduj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBledy">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista błędów</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
